src/game_api/script/lua_backend.hpp

```diff
@@ -94,6 +94,8 @@ enum class ON
     PRE_LOAD_SCREEN,
     POST_LOAD_SCREEN,
     DEATH_MESSAGE,
+    PRE_UPDATE,
+    UPDATE,
```

Comment:
Shouldn't this be `POST_UPDATE`?

Reply:
Dreagu requested change from everyframe to update, but makes sens to make it post_update